daemon/src/management/FileDescriptor.h

```diff
@@ -40,6 +41,11 @@ struct FileDescriptor
 {
     const static unsigned char M_READ = 1;
     const static unsigned char M_WRITE = 2;
+    const static unsigned char M_CREATE = 4;
+    const static unsigned char M_CREATE_PERM_R = 8;
+    const static unsigned char M_CREATE_PERM_W = 16;
+    const static unsigned char M_CREATE_PERM_X = 32;
+
```

Comment:
I assume those are fully compatibile with flags like O_CREATE, O_WRONLY etc.?

Reply:
No, but there is a wrapper within open function which converts linux flags into these above.